frontend/fingertips-frontend/components/molecules/FilterSummaryPanel/FilterSummaryPanel.test.tsx

```diff
@@ -0,0 +1,93 @@
+import { render, screen } from '@testing-library/react';
+import {
+  FilterSummaryPanel,
+  FilterSummaryPanelProps,
+} from '@/components/molecules/FilterSummaryPanel/index';
+import { SearchParams } from '@/lib/searchStateManager';
+import { generateIndicatorDocument } from '@/lib/search/mockDataHelper';
+import { nhsPrimaryCareNetworksAreaType } from '@/lib/areaFilterHelpers/areaType';
+import { userEvent, UserEvent } from '@testing-library/user-event';
+import { LoaderContext } from '@/context/LoaderContext';
+
+const mockPath = 'some-mock-path';
+const mockReplace = jest.fn();
+
+const mockGetIsLoading = jest.fn();
+const mockSetIsLoading = jest.fn();
+const mockLoaderContext: LoaderContext = {
+  getIsLoading: mockGetIsLoading,
+  setIsLoading: mockSetIsLoading,
```

Comment:
Can change these to 
```
 getIsLoading: jest.fn(),
  setIsLoading: jest.fn(),
```
If you don't need to spy or change the return values in the tests